__tests__/About.snapshot.tsx

```diff
@@ -39,8 +45,8 @@ describe('Component About - test', () => {
   //snapshot test
   test('About - snapshot', () => {
     const state = {
-      navigation: null,
-      route: null,
+      navigation: {} as StackScreenProps<any>['navigation'],
```

Comment:
Sweet!   So these are like...  empty instances of the type `StackScreenProps<any>['navigation']`?!

Reply:
I have only a loose understanding of TypeScript, but these changes look great to me!

Reply:
Yes, you can initialize with an empty object, but declaring eyyyyy, here only accept this kind of value. Way better than `null`.